### PR TITLE
Trying to get fewer master branch training errors

### PR DESCRIPTION
**Pull request by olafthiele**
As more and more people simply clone the master branch, maybe this steers them towards using a released branch.

### CODE REVIEW

**Comment by olafthiele** [comment]
I updated as discussed

File: doc/TRAINING.rst [hunk 1]
```diff
@@ -12,11 +12,13 @@ Prerequisites for training a model
 Getting the training code
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Clone the DeepSpeech repository:
+Clone the latest released stable branch from Github (e.g. 0.8.2, check `here <https://github.com/mozilla/DeepSpeech/releases>`_ for current version) to start with as the master branch is under active development and might produce errors:
```
Comment (lissyx) [inline doc/TRAINING.rst:15]: nit: "the master branch provides no guarantee regarding its workable state, though we are trying to keep it as stable as possible ; breakages can happen and feedback is welcome."

**Review by lissyx — APPROVED** [review]
@reuben how do you feel about the wording / my suggestion?

**Comment by olafthiele** [comment]
I understand what @lissyx wants to state, but it sounds a bit overly complicated to me. You don't have to justify yourselves in the first paragraph, this is a great project :-) This PR is just meant to reduce the number of people who don't read and simply copy from the docs and then file bug reports. How about we just put

`Clone the latest released stable branch from Github (e.g. 0.8.2, check here):`

and leave all the rest out? We state to use the master branch in the next sentence and people who want to start right away can do so. And those who do have a programming background and want to contribute know to look into the contributing file. @reuben?

**Comment by reuben** [comment]
Agree with Olaf, just telling people to clone the latest stable version should be fine. No amount of extra warning will prevent some people from messing up, we just want to have *some* explicit mention of using the stable version in the training docs.

**Review by reuben — APPROVED** [review]

**Comment by lissyx** [comment]
Fine, if you both agree it's enough let's ship it!

**Comment by olafthiele** [comment]
Sorry @lissyx, I thought we were still discussing, it currently is the old version, I didn't change it yet. Will do and see whether I can attach it to this pr. Haven't changed a merged one yet.

**Comment by lissyx** [comment]
> Sorry @lissyx, I thought we were still discussing, it currently is the old version, I didn't change it yet. Will do and see whether I can attach it to this pr. Haven't changed a merged one yet.

my bad, just open a new PR and fix it :)

**Pull request merged**